src/Microsoft.VisualStudio.Sdk.TestFramework/SdkTestUtilities.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if NETFRAMEWORK || WINDOWS
+
+using System.IO;
+using ComAsyncServiceProvider = Microsoft.VisualStudio.Shell.Interop.COMAsyncServiceProvider.IAsyncServiceProvider;
+using OleServiceProvider = Microsoft.VisualStudio.OLE.Interop.IServiceProvider;
+
+namespace Microsoft.VisualStudio.Sdk.TestFramework;
+
+/// <summary>
+/// Test utilities for VS SDK tests.
+/// </summary>
+public static class SdkTestUtilities
+{
+    /// <summary>
+    /// Initializes an instance of an <see cref="AsyncPackage"/>.
+    /// </summary>
+    /// <typeparam name="T">The package to initialize.</typeparam>
+    /// <param name="logger">An optional logger that will receive progress updates.</param>
+    /// <returns>The loaded package.</returns>
+    public static async Task<T> LoadPackageAsync<T>(Action<string>? logger = null)
+        where T : AsyncPackage, new()
```

Comment:
Technically all the shell actually requires is a package implements IAsyncLoadablePackageInitialize, but requiring it actually be MPF AsyncPackage is reasonable too

Reply:
Ya, I think it makes it more obvious what type the user should put there.